docs/source/en/quick_tour.md

```diff
@@ -101,3 +101,23 @@ curl 127.0.0.1:8080/predict \
     -d '{"inputs":"I like you."}' \
     -H 'Content-Type: application/json'
 ```
+
+## Batching
+
+You can send multiple inputs in a batch. For example, for embeddings
+
+```bash
+curl 127.0.0.1:8080/embed \
+    -X POST \
+    -d '{"inputs":[["Today is a nice day"], ["I like you"]]}' \
```

Comment:
```suggestion
    -d '{"inputs":["Today is a nice day", "I like you"]}' \
```

`/embed` do not support pairs right now.

Reply:
This is not a pair, but a batch of single strings no?